packages/venia-concept/src/index.js

```diff
@@ -20,6 +21,21 @@ import { registerSW } from './registerSW';
 const { BrowserPersistence } = Util;
 const apiBase = new URL('/graphql', location.origin).toString();
 
+const language = STORE_VIEW_LOCALE;
+let messages;
+
+try {
+    messages = require(`@magento/venia-ui/lib/i18n/${language.replace(
+        '-',
+        '_'
+    )}.json`);
+} catch (err) {
+    console.warn(
+        `Unable to load translation file. Loading 'en_US' instead. \n${err}`
```

Comment:
I would be inclined to include which file path was attempted to be loaded here to help with debugging.